cloudbuild/tag_latest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Could you say more about this script?

Why we can't just tag built image with the `latest` tag and push it using `cloudbuild-tag.yaml` file? https://github.com/GoogleCloudPlatform/marketplace-testrunner/blob/528059349a48c2ce17a0aa62dc6acdd4fb8bb695/cloudbuild-tag.yaml#L8-L12

Reply:
I updated the design doc for CI/CD. Copy-n-pasting the content of that section here:

We should tag the most recent release with the latest tag as well. Doing that in `cloudbuild-tag.yaml` _blindly_ fails the following 2 scenarios:
- (1) We can cut two separate minor version releases, for example `3.1.2` and `3.2.3`. Even if we cut `3.1.2` after, only `3.2.3` should be tagged as `latest`.
- (2) It's unlikely but possible for two tag-triggered releases to happen concurrently. We have a race condition.

While we _can't completely eliminate (2)_, we can get almost there and eliminate (1) by doing the following:
- Before setting the `latest` tag, list all tags in the repo. Pick the latest one.
- If that latest one from the repo is after the current tag, based on SemVer, pull the remote image and tag that with latest. We have to do this because we can't conditionally choose whether we want to push an image.
- Otherwise, use `latest` on the current tag.

Reply:
Could you include this note into the script, please?

Reply:
Added comment about what the script does.